javaparser-core-testing/src/test/java/com/github/javaparser/utils/VisitorSetTest.java

```diff
@@ -162,7 +162,7 @@ void visitorSetToString() {
         CompilationUnit x2 = parse("class Y{}");
         set.add(x2);
 
-        String[] validStrings = new String[]{"[" + x1 + "," + x2 + "]", "[" + x2 + "," + x1 + "]"};
+        String[] validStrings = new String[]{"[" + x1 + ", " + x2 + "]", "[" + x2 + ", " + x1 + "]"};
```

Comment:
I think it is this pattern that should be modified by using for example the join functionality in streams.

> for (EqualsHashcodeOverridingFacade facade : innerList) {
            sb.append(facade.overridden.toString() + ", ");
        }
        return sb.replace(sb.length() - 2, sb.length(), "]").toString();

Reply:
To clarify sir, when you say the pattern that should be modified, are you referring to the `StringBuilder` for loop in the `toString` method or the way `validStrings` are created in the test? Or perhaps both should be modified?

Reply:
Hi @flanbino both should be modified in particular i refer to this code that add a separator and finally replace the last separator.

Reply:
Hello again, I've made the changes requested